src/index.md

```diff
@@ -1,5 +1,6 @@
 ---
 title: Flutter - Beautiful native apps in record time
+layout: landing
 homepage: true
 hide_title: true
```

Comment:
- `page.hide_title` isn't used anywhere AFAICT, so I'd suggest removing it.
- `page.homepage` is used in `head.html`, but we might want to trigger on `page.layout == "landing"` instead.

Reply:
the way it's being used right now is to control the HTML title, I think that's a specific thing for the homepage, since other pages could theoretically use landing.html (like the showcase page).